marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -588,6 +680,8 @@ const Flows: React.FC<Props> = ({
     } else {
       onCheckboxChange("default", step.stepName, step.stepNumber, step.stepDefinitionType, flowNameCheck, step.stepId, step.sourceFormat);
     }
+    // if (getUserPreferencesLS() && getUserPreferencesLS()?.loadSelectedStepsUser) getLocalStorageDataUser();
```

Comment:
Let's remove the comments if the code is not needed anymore.

Reply:
Thanks, done.